components/infobox/commons/infobox_person.lua

```diff
@@ -267,6 +267,11 @@ function Person:_setLpdbData(args, links, status, personType)
 		lpdbData.extradata['earningsin' .. year] = earningsOfYear
 	end
 
+	args.team1 = team
+	for teamKey, otherTeam in Table.iter.pairsByPrefix(args, 'team') do
+		lpdbData.extradata[teamKey] = args[teamKey .. 'link'] or otherTeam
+	end
```

Comment:
* `team1` is already stored outside extradata
* I would prefer to store team templates instead of non-normalized names/links

Reply:
```suggestion
	args.team1 = team
	for teamKey, teamName, index in Table.iter.pairsByPrefix(args, 'team') do
		if index > 1 then
			lpdbData.extradata[teamKey] = (mw.ext.TeamTemplate.raw(args[teamKey .. 'link'] or teamName) or {}).page
		end
	end
```

Reply:
Also FWIW, `teamXlink` has no use over `teamX`, it's simply used as a legacy fallback

Reply:
I don't want the template, I want the page name.
Why must it be made harder to iterate over?

Reply:
Because pagename are a hell of themself to work with.
* Uppercase vs lowercase first letters
* Underscore vs space as separators
* Redirected vs not redirected

In general we are moving away from using pagenames when it comes to teams. We have team templates, which contains the page of the team, and several more attributes. It's not harder to query by team template compared to team pagename, frankly it's easier, because of the above issues.

Reply:
I simply resolve the redirect then all those issues are avoided

Reply:
With team templates I would have to get the page name, just to resolve the redirect anyway

Reply:
Anyways, in my opinion it doesn't seem necessary to add code to remove the first team, it just makes it more annoying to iterate over, especially since you are suggesting storing a different value to the first team

Reply:
At the very least the additional teams should be able to be parsed the same way as the primary team, but that is not the case in your suggestion

Reply:
@Rathoz

Reply:
I've added a commit 6f938e613ff36941e702241bb084de6183389129 now to sanitise the teams, but if you skip the first entry then the equivalent of it isn't `team`

Reply:
Sorry for late reply. Friday ended up weird.

> With team templates I would have to get the page name, just to resolve the redirect anyway

You can query directly off the TeamTemplate. Team have this information stored (as the DB field `template`)

> if you skip the first entry then the equivalent of it isn't team

It's equivalent of `teamtemplate`, not `team`. However the current `team` is deprecated due to the exact storage value is undefined storage.

Reply:
>but only one redirected-pagename

This is technically speaking not true, since a team can be a historical setup (ie has a new (sub-)teamtemplate & pagename due to name change) without having a page on the wiki (and hence cannot have a redirect).

> The reason I primarily use pagename, is that a team can have many templates (subtemplates),

You can always use `Team.queryHistoricalNames(...)` to get fetch all. 

Example 
https://github.com/Liquipedia/Lua-Modules/blob/a88b0a1efa6043a76ed2ab84a8d2aab8426e1b2d/components/earnings/commons/earnings_base.lua#L91
https://github.com/Liquipedia/Lua-Modules/blob/a88b0a1efa6043a76ed2ab84a8d2aab8426e1b2d/components/earnings/commons/earnings_base.lua#L109-L113

Reply:
I see your point

I've committed the 1st-team skip 3b39ef6